app/assets/stylesheets/app.scss

```diff
@@ -50,7 +50,7 @@ a {
 }
 
 #breaking-news-container {
-  min-height: 71px;
+  min-height: 51px;
```

Comment:
51 ?

Reply:
Minimum height of braking news to avoid CLS